### PR TITLE
*: fix Polish translation

### PR DESCRIPTION
**Pull request by orgiele**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Use the correct Polish form. I did not know how to format my commit message; any suggestions are welcome.

### CODE REVIEW

File: pages.pl/common/clang-cpp.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # clang-cpp
 
-> Te polecenie jest aliasem `clang++`.
+> To poleceniejest aliasem `clang++`.
```
Comment (DawidStruzik) [inline pages.pl/common/clang-cpp.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/common/clojure.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # clojure
 
-> Te polecenie jest aliasem `clj`.
+> To poleceniejest aliasem `clj`.
```
Comment (DawidStruzik) [inline pages.pl/common/clojure.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/common/cola.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # cola
 
-> Te polecenie jest aliasem `git-cola`.
+> To poleceniejest aliasem `git-cola`.
```
Comment (DawidStruzik) [inline pages.pl/common/cola.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/common/llvm-gcc.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # llvm-gcc
 
-> Te polecenie jest aliasem `clang`.
+> To poleceniejest aliasem `clang`.
```
Comment (DawidStruzik) [inline pages.pl/common/llvm-gcc.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/common/llvm-nm.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # llvm-nm
 
-> Te polecenie jest aliasem `nm`.
+> To poleceniejest aliasem `nm`.
```
Comment (DawidStruzik) [inline pages.pl/common/llvm-nm.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/common/llvm-objdump.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # llvm-objdump
 
-> Te polecenie jest aliasem `objdump`.
+> To poleceniejest aliasem `objdump`.
```
Comment (DawidStruzik) [inline pages.pl/common/llvm-objdump.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/common/llvm-strings.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 # llvm-strings
 
-> Te polecenie jest aliasem `strings`.
+> To poleceniejest aliasem `strings`.
```
Comment (DawidStruzik) [inline pages.pl/common/llvm-strings.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/common/mscore.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 # mscore
 
-> Te polecenie jest aliasem `musescore`.
+> To poleceniejest aliasem `musescore`.
```
Comment (DawidStruzik) [inline pages.pl/common/mscore.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/common/pio-init.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 # pio-init
 
-> Te polecenie jest aliasem `pio project`.
+> To poleceniejest aliasem `pio project`.
```
Comment (DawidStruzik) [inline pages.pl/common/pio-init.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/common/piodebuggdb.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 # piodebuggdb
 
-> Te polecenie jest aliasem `pio debug`.
+> To poleceniejest aliasem `pio debug`.
```
Comment (DawidStruzik) [inline pages.pl/common/piodebuggdb.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/common/r2.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
 # r2
 
-> Te polecenie jest aliasem `radare2`.
+> To poleceniejest aliasem `radare2`.
```
Comment (DawidStruzik) [inline pages.pl/common/r2.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/common/tldrl.md [hunk 12]
```diff
@@ -1,6 +1,6 @@
 # tldrl
 
-> Te polecenie jest aliasem `tldr-lint`.
+> To poleceniejest aliasem `tldr-lint`.
```
Comment (DawidStruzik) [inline pages.pl/common/tldrl.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/common/tlmgr-arch.md [hunk 13]
```diff
@@ -1,6 +1,6 @@
 # tlmgr-arch
 
-> Te polecenie jest aliasem `tlmgr platform`.
+> To poleceniejest aliasem `tlmgr platform`.
```
Comment (DawidStruzik) [inline pages.pl/common/tlmgr-arch.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/common/vi.md [hunk 14]
```diff
@@ -1,6 +1,6 @@
 # vi
 
-> Te polecenie jest aliasem `vim`.
+> To poleceniejest aliasem `vim`.
```
Comment (DawidStruzik) [inline pages.pl/common/vi.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/linux/ncal.md [hunk 15]
```diff
@@ -1,6 +1,6 @@
 # ncal
 
-> Te polecenie jest aliasem `cal`.
+> To poleceniejest aliasem `cal`.
```
Comment (DawidStruzik) [inline pages.pl/linux/ncal.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/windows/cinst.md [hunk 16]
```diff
@@ -1,6 +1,6 @@
 # cinst
 
-> Te polecenie jest aliasem `choco install`.
+> To poleceniejest aliasem `choco install`.
```
Comment (DawidStruzik) [inline pages.pl/windows/cinst.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/windows/clist.md [hunk 17]
```diff
@@ -1,6 +1,6 @@
 # clist
 
-> Te polecenie jest aliasem `choco list`.
+> To poleceniejest aliasem `choco list`.
```
Comment (DawidStruzik) [inline pages.pl/windows/clist.md:3]: Missing space. Should be "polecenie jest".

File: pages.pl/windows/cuninst.md [hunk 18]
```diff
@@ -1,6 +1,6 @@
 # cuninst
 
-> Te polecenie jest aliasem `choco uninstall`.
+> To poleceniejest aliasem `choco uninstall`.
```
Comment (DawidStruzik) [inline pages.pl/windows/cuninst.md:3]: Missing space. Should be "polecenie jest".

**Review by DawidStruzik — COMMENTED** [review]
Hi, here are my comments.

File: pages.pl/common/clang-cpp.md (hunk 1 above)
Comment (orgiele) [reply pages.pl/common/clang-cpp.md]: Fixed, sorry :)

File: pages.pl/common/clojure.md (hunk 2 above)
Comment (orgiele) [reply pages.pl/common/clojure.md]: Fixed, sorry :)

File: pages.pl/common/cola.md (hunk 3 above)
Comment (orgiele) [reply pages.pl/common/cola.md]: Fixed, sorry :)

**Comment by orgiele** [comment]
It should be ok now

**Review by DawidStruzik — APPROVED** [review]
LGTM

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by chanibal — APPROVED** [review]
LGTM

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**